### PR TITLE
switch to using docker for dist server instead of simple http server

### PR DESCRIPTION
**Pull request by ryanhobsonsmith**
use docker/nginx for test production server instead of python's SimpleHTTP server.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -8,7 +8,7 @@
     "test": "",
     "build": "npm run clean; webpack --env=prod --colors",
     "clean": "rm -rf dist",
-    "dist:server": "pushd dist; python -m SimpleHTTPServer; popd"
+    "dist:server": ". ./scripts/dist_server.sh"
```
Comment (stuartsan) [inline package.json]: ~~Is the leading dot intentional?~~ NVM I get it, hadn't seen that before!

**Review by stuartsan — APPROVED** [review]
+1 with the side note that it might be worth calling out in the README that docker is a dev dependency

File: package.json (hunk 1 above)
Comment (shazow) [reply package.json]: The leading dot is equivalent to `source`, which I don't think is what we want (it's for injecting variables into our current environment)? If it's executable, then just `./script/dist_server.sh` should do the trick, no?

Comment (stuartsan) [reply package.json]: [ defer, no idea, I thought I tested this and saw that it was needed but now can't repro... ]

**Comment by stuartsan** [comment]
@ryanhobsonsmith , just brought this up to date with master. Per @shazow 's comments above and my testing and seeing the script working as desired without the leading dot to source it, I removed that. If it was there for some particular reason LMK, I'll put it back!

Merging.

**Pull request merged**